README.md

```diff
@@ -237,6 +237,21 @@ flavor:
 * `INSTANCE_MYSQL_URL` If using an external mysql database, set its url here
 * `INSTANCE_MONGO_URL` If using an external mongo database, set its url here
 
+### Open edX specific settings
+
+* `DEFAULT_OPENEDX_RELEASE` Set this to a release tag like 
+  `"named-release/dogwood"` to specify the default release of Open edX to use.
+  This setting becomes the default value for `edx_platform_version`,
+  `forum_version`, `notifier_version`, `xqueue_version`, and `certs_version` so
+  it should be a git branch or tag that exists in all of those repositories.
+* `DEFAULT_CONFIGURATION_REPO_URL` The repository containing the Open edX
+  ansible scripts to use. Defaults to
+  `'https://github.com/edx/configuration.git'`
```

Comment:
@bradenmacdonald No need for the single quotes here. Also (nit): "ansible" ⟶ "Ansible". Sentence is missing a full stop.